### PR TITLE
Dedup->dedupe->deduplicate

### PR DESCRIPTION
**Pull request by peternewman**
See also notes in #1602

### CODE REVIEW

**Comment by larsoner** [comment]
Travis is not happy

**Comment by sebweb3r** [comment]
Travis will never be happy with that. Informal requires both error and replacement to be in aspell. None of the three words is in aspell.
http://app.aspell.net/lookup?dict=en_US;words=dedup%0D%0Adedupe%0D%0Adeduplicate

**Comment by larsoner** [comment]
> Travis will never be happy with that. Informal requires both error and replacement to be in aspell. None of the three words is in aspell.

Travis will do whatever we say :) Concretely, I'm fine with modifying the test suite to have exceptions (`# do not check aspell for this set of words`) in cases like these if it's justified to do so

**Comment by sebweb3r** [comment]
Ok, let me rephrase it. Without bending travis, it won't accept it.

This basically follows this trend:
https://github.com/codespell-project/codespell/issues/1711

**Comment by lurch** [comment]
I suspect these commits need squashing? :wink:

**Comment by bl-ue** [comment]
@peternewman can I merge this PR? (**SQUASH definitely!!**)

**Comment by DimitriPapadopoulos** [comment]
@peternewman It should be possible to merge this pull request if you resolve the conflicts and put words missing from _aspell_ under `codespell_lib/tests/data`.
